spec/views/hyrax/base/_attributes.html.erb_spec.rb

```diff
@@ -14,7 +14,7 @@
       description_tesim: description
     }
   end
-  let(:ability) { nil }
+  let(:ability) { double(admin?: true) }
```

Comment:
This seems like a significant change in the test.  Can you say more about the need for this change?

Reply:
because I added the user ability testing to the method that displays the collection links, it now accesses a method that checks for admin? Since the actual search was tested elsewhere, This view doesn't need to test the ability logic.